### PR TITLE
Bug/fz app folder strucute RIUI-1792

### PR DESCRIPTION
**Pull request by fazanki**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by Mo-Lala84** [comment]
Approved and looks in line with the agreed structure

File: src/app/containers/app/app.component.spec.ts [hunk 1]
```diff
@@ -1,16 +1,20 @@
 import { TestBed, async } from '@angular/core/testing';
 import { AppComponent } from './app.component';
-import { LoggerService } from './logger/logger.service';
-
+import { LoggerService } from '../../service/logger/logger.service';
+import { RouterModule } from '@angular/router';
```
Comment (pardeepbasi) [inline src/app/containers/app/app.component.spec.ts]: https://angular.io/api/router/testing/RouterTestingModule probably best to use the router testing module within tests

File: src/app/service/logger/console-logger.service.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ export class ConsoleLoggerService implements Logger {
   /* tslint:disable:no-console */
   get info() {
     if (isDebugMode !== 0 && isDebugMode !== 1 && isDebugMode !== 2) {
-      return console.info.bind(console, [user, now]);
+      return console.warn.bind(console, [user, now]);
```
Comment (pardeepbasi) [inline src/app/service/logger/console-logger.service.ts]: why is this change needed? essentially that would mean all logger.info() calls are executed as warnings

Comment (fazanki) [reply src/app/service/logger/console-logger.service.ts]: console info is not allowed as lint is falling

File: src/app/containers/app/app.component.spec.ts (hunk 1 above)
Comment (fazanki) [reply src/app/containers/app/app.component.spec.ts]: done

File: src/app/service/logger/console-logger.service.ts (hunk 2 above)
Comment (fazanki) [reply src/app/service/logger/console-logger.service.ts]: done
